libs/mark-flow-ui/src/lib/screen_orientation.ts

```diff
@@ -0,0 +1,15 @@
+import { MachineConfig } from '../config/types';
+
+interface ScreenOrientationReturnType {
+  isLandscape: boolean;
+  isPortrait: boolean;
+}
+
+export function screenOrientation(
+  machineConfig: MachineConfig
+): ScreenOrientationReturnType {
+  const isLandscape = machineConfig.screenOrientation === 'landscape';
```

Comment:
This makes me wonder: will we support a landscape mode for RAVE as well?
I might be doing some cleanup in VxMark later to remove landscape support, so want to make sure I don't end up removing anything that will be useful later

Reply:
I think the idea is we'll _only_ have landscape. Tagging @mattroe for confirmation.

Reply:
Ah, good to know! I guess when the time comes, I can just focus on the pieces specific to the machine UI that are no longer needed. Can leave all orientation logic within libs/mark-flow-ui intact